src/messages/index.ts

```diff
@@ -0,0 +1,42 @@
+import BaseApi from '../baseApi';
```

Comment:
You are missing imports, inits and exports in index.ts. Also add a [x] to Readme.md for messages.